deploy/scripts/src/txma/test.ts

```diff
@@ -68,9 +68,42 @@ const profiles: ProfileList = {
       startRate: 1,
       timeUnit: '1s',
       preAllocatedVUs: 1,
-      maxVUs: 1500,
+      maxVUs: 2000,
+      stages: [
+        { target: 1000, duration: '10m' } // Ramp up to 30 iterations per second in 60 minutes
+      ],
+      exec: 'sendEvent'
+    },
+    sendEventLoadTest5: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 3000,
+      stages: [
+        { target: 1500, duration: '10m' } // Ramp up to 30 iterations per second in 60 minutes
```

Comment:
```suggestion
        { target: 1500, duration: '10m' } // Ramp up to 1500 iterations per second in 10 minutes
```
Update comments to match values

Reply:
Done